pkg/agent/proxy/proxier_windows.go

```diff
@@ -25,22 +26,45 @@ import (
 // The rules for traffic from local Pod to LoadBalancer Service are the same with rules for Cluster Service.
 // For the LoadBalancer Service traffic from outside, specific rules are install to forward the packets
 // to the host network to let kube-proxy handle the traffic.
-func (p *proxier) installLoadBalancerServiceFlows(groupID binding.GroupIDType, svcIP net.IP, svcPort uint16, protocol binding.Protocol, affinityTimeout uint16) error {
-	if err := p.ofClient.InstallServiceFlows(groupID, svcIP, svcPort, protocol, affinityTimeout); err != nil {
-		return err
+// Note that, NodePort Service is not supported on Windows currently.
+func (p *proxier) installLoadBalancerService(groupID binding.GroupIDType, loadBalancerIPStrings []string,
```

Comment:
If the implementations for LoadBalancerService are the same on Linux and Windows, maybe move the function to proxier.go?

Reply:
I think it is ok, `Windows` has one more function, `Linux` will just hit a ni function. I think it is ok.